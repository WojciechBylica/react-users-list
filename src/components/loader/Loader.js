import './styles.scss';

const Loader = () => <p className="loader">loading...</p>;

export default Loader;
