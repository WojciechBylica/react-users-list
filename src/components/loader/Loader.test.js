import React from 'react';
import renderer from 'react-test-renderer';
import Loader from '.';

it('should render text "loading..."', () => {
  const tree = renderer.create(<Loader />);
  expect(tree).toMatchSnapshot();
});
