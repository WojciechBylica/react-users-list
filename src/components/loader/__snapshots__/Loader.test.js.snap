// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render text "loading..." 1`] = `
<p
  className="loader"
>
  loading...
</p>
`;
